File: src/main/java/hilfling/backend/hilfling/controllers/PhotoTagController.java
package hilfling.backend.hilfling.controllers;

import hilfling.backend.hilfling.model.PhotoTag;
import hilfling.backend.hilfling.service.PhotoTagService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.*;

@RestController
@RequestMapping("/api/v1/photo_tags")
public class PhotoTagController extends GenericBaseControllerImplementation<PhotoTag> {
    @Autowired
    private PhotoTagService service;
    public PhotoTagService getService() {
        return service;
    }
}

File: src/main/kotlin/no/fg/hilflingbackend/controller/PhotoTagController.kt
package no.fg.hilflingbackend.controller

import no.fg.hilflingbackend.dto.PhotoTagDto
import no.fg.hilflingbackend.model.PhotoTag
import no.fg.hilflingbackend.repository.PhotoTagRepository
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController()
@RequestMapping("/phototags")
open class PhotoTagController(override val repository: PhotoTagRepository) : BaseController<PhotoTag, PhotoTagDto>(repository)
